multihost_runner.py

```diff
@@ -133,29 +133,24 @@ def write_kill_script(script_dir, kill_processes_script_name):
 
 def kill_existing_processes_str():
   return """#!/bin/bash
-_TPU_VERSION_NAME=${1}
+_TPU_VERSION_NAME="${1}"
 device_name="accel"
 if [[ "${_TPU_VERSION_NAME}" =~ ^v5.* ]]; then
   device_name="vfio/"
 fi
-echo -e "Searching for existing processes on device ${device_name}..."
-pids=$(sudo lsof -t -w /dev/${device_name}*)
-pids_unique=$(printf '"'"'%s\\n'"'"' "${pids[@]}" | sort -u)
-pids_unique=($(printf '"'"'%s\\n'"'"' "${pids_unique[@]}" | grep -v '"'"'^[[:space:]]*$'"'"'))
-num_pids=${#pids_unique[@]}
-
-if [[ ! ${num_pids} -eq 0 ]] 
-then
-    echo "Found ${num_pids} unique PID(s) already running on your TPUs, killing now..."
-    for pid in "${pids_unique[@]}"; do
-        echo -e "Killing process ${pid}..."
-        kill -9 "${pid}"
-        tail --pid=$pid -f /dev/null
-        echo -e "Existing process ${pid} on your TPU was killed successfully!"
-    done
-    echo "All existing processes killed, so your TPU is ready to use!"
+echo "Searching for existing processes on device ${device_name}..."
+pids=$(sudo lsof -t /dev/${device_name}* | sort -u)
```

Comment:
Was the -w option removed intentionally?

Reply:
Ya Phillip thinks there is no reason for it. I believe -w suppresses warning messages, but I haven't seen any warning messages with or without -w in either case of a clean TPU or needing to kill PIDs

Reply:
Great, thanks Matt